kong/plugins/oauth2/migrations/postgres.lua

```diff
@@ -155,5 +155,28 @@ return {
         end
       end
     end
+  },
+  {
+    name = "2017-7-27-convert_redirect_uri_to_pattern",
+    up = function(_, _, factory)
+      local schema = factory.oauth2_credentials.schema
+
+      local rows, err = factory.oauth2_credentials.db:find_all("oauth2_credentials", nil, schema)
+      if err then
+        return err
+      end
+
+      for _, row in ipairs(rows) do
+        local converted = {}
+        for _, uri in ipairs(row.redirect_uri) do
```

Comment:
would prefer to see an integer loop here, as `_` is often used (and treated by our linter) as a discard character, making it semantically meaningless.